ui.frontend/src/main/styles/commerce/_breadcrumb.scss

```diff
@@ -0,0 +1,9 @@
+.cmp-breadcrumb__list {
```

Comment:
Is there any particular reason why you didn't use https://github.com/adobe/aem-cif-guides-venia/blob/main/ui.frontend/src/main/styles/_breadcrumb.scss?

Reply:
I put it there because the css for the navigation is also there (https://github.com/adobe/aem-cif-guides-venia/blob/main/ui.frontend/src/main/styles/commerce/_navigation.scss) and in the parent folder.

Reply:
@cjelger We should put these styles at the location @herzog31 pointed to. This component is, after all, inheriting from the Core Components Breadcrumb component and should re-use the same styles.

Reply:
@herzog31 @dplaton I moved the css to the existing file.